src/CMakeLists.txt

```diff
@@ -96,6 +96,8 @@ target_link_libraries(osp-magnum-deps PUBLIC
     toml11
     spdlog
     longeron
+    RmlCore
+    freetype
```

Comment:
I don't believe that you need to explicitly link to freetype. RmlCore already does that for you.